rundeck_exporter.py

```diff
@@ -163,6 +164,19 @@ class RundeckMetricsCollector(object):
                             nargs='+',
                             required=False
                             )
+    args_parser.add_argument('--rundeck.projects.executions.job-option-labels',
+                             dest='rundeck_projects_executions_job_option_labels',
+                             help='A space-separated list of specific job option keys to be exposed as dedicated Prometheus labels with opt_ prefix (e.g., cluster_name). If empty and --rundeck.projects.executions.export-job-options is enabled, all job options will be exported.',
+                             default=getenv('RUNDECK_PROJECTS_EXECUTIONS_JOB_OPTION_LABELS', '').split(),
+                             nargs='*',
+                             required=False
+                             )
+    args_parser.add_argument('--rundeck.projects.executions.export-job-options',
```

Comment:
What about using snake_case and also changing it to something like `--rundeck.projects.executions.job_options_as_labels`?